test/parallel/test-child-process-require.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var assert = require('assert');
+var common = require('../common');
+var child_process = require('child_process');
+
+assert.equal(typeof child_process.ChildProcess, 'function');
```

Comment:
we should flesh out this test file a bit – make sure we can instantiate it, call spawn on it, and call kill on it.